app/src/main/res/layout-land/fragment_phrases.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/phrases_container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:columnCount="3"
```

Comment:
Is the only thing different between these buckets (landscape, portrait, tablet) the column and row counts?

Reply:
That's what I'm thinking, yeah - move them into `integers.xml` and just make different buckets

Reply:
We're going to handle this in an upcoming PR where we add a couple of more buckets.